file/types.go

```diff
@@ -253,12 +253,12 @@ type FTarget struct {
 
 // id is used for sorting.
 func (t FTarget) id() string {
```

Comment:
Since the `id` function no longer returns the ID primarily, its name is misleading and it will lead to bugs.

Please rename:
- func `id()` to `sortKey()`
- interface `id` to `sortable`
- func `compareID` to `compareOrder`